README.md

```diff
@@ -0,0 +1,49 @@
+**![](https://img.shields.io/badge/-microverse-yellow)**
+
+# Displaying And Inputting Data (Login page)
```

Comment:
I think the project is not `Login page` it's actually a `signup page`
![Screenshot from 2020-10-27 00-19-15](https://user-images.githubusercontent.com/56886064/97235382-6a1d7180-17eb-11eb-837e-33ac478c073f.png)